esti/main_test.go

```diff
@@ -15,7 +18,10 @@ import (
 	"github.com/treeverse/lakefs/pkg/testutil"
 )
 
-type Booleans []bool
+type (
+	Booleans   []bool
+	arrayFlags []string
```

Comment:
Maybe I've missed it - but why do you need to wrap it in a type with dedicated functions?
I couldn't find any user for Set, or String - which interface is this suppose to implement?

Reply:
Not really TBH

Reply:
Yes, the `flag.Var` function, gets a "value" interface as its first parameter and this interface should implement 	`String() string` and `Set(string) error` functions